README.md

```diff
@@ -20,7 +20,10 @@ To learn how to contribute to the MetaMask project itself, visit our [Internal D
 - Build the project to the `./dist/` folder with `npm run dist`.
 - Optionally, to start a development build (e.g. with logging and file watching) run `npm start` instead.
     - To start the [React DevTools](https://github.com/facebook/react-devtools) and [Redux DevTools Extension](http://extension.remotedev.io)
-      alongside the app, use `npm run start:dev`. You'll need to install the Redux DevTools to access Redux state logs.
+      alongside the app, use `npm run start:dev`.
+      - React DevTools will open in a separate window; no browser extension is required
+      - Redux DevTools will need to be installed as a browser extension. Open the Redux Remote DevTools (shown in the context menu and in the extension menu) to access Redux state logs.
```

Comment:
I guess you could be very explicit and explain how to access the context menu / extension menu here.

> Open the Redux Remote Devtools. This can be done by either right clicking within the web browser to bring up the context menu, expanding the Redux DevTools panel and clicking `Open Remote DevTools` OR clicking the Redux DevTools extension icon and clicking `Open Remote DevTools`.

Reply:
Good suggestion! I've already merged this PR, but if you wanted to put up another PR with that change I'd be happy to merge it.

Reply:
Done! https://github.com/MetaMask/metamask-extension/pull/6917